pipelines/common/pom.xml

```diff
@@ -120,7 +120,11 @@
     <dependency>
       <groupId>ca.uhn.hapi.fhir</groupId>
       <artifactId>hapi-fhir-converter</artifactId>
-      <version>${hapifhirVersion}</version>
+    </dependency>
+    <!-- This library is used for caching of resources -->
+    <dependency>
+      <groupId>ca.uhn.hapi.fhir</groupId>
+      <artifactId>hapi-fhir-caching-caffeine</artifactId>
```

Comment:
Do you know what piece of our code triggers this dependency requirement?

Reply:
This dependency is needed for the `ViewApplicator` which validates the fhirPath using the `IFhirPath` class. I have added the error that will be thrown in the description of this PR, when this dependency is not added. Adding it here again - 

```java.lang.RuntimeException: HAPI-2200: No Cache Service Providers found. Choose between hapi-fhir-caching-caffeine (Default) and hapi-fhir-caching-guava (Android)

	at ca.uhn.fhir.sl.cache.CacheFactory.getCacheProvider(CacheFactory.java:41)
	at ca.uhn.fhir.sl.cache.CacheFactory.build(CacheFactory.java:46)
	at org.hl7.fhir.r4.hapi.ctx.HapiWorkerContext.<init>(HapiWorkerContext.java:65)
	at org.hl7.fhir.r4.hapi.fluentpath.FhirPathR4.<init>(FhirPathR4.java:31)
	at com.google.fhir.analytics.view.ViewApplicator.<init>(ViewApplicator.java:66)
	at com.google.fhir.analytics.view.ViewApplicatorTest.applyViewOnResource(ViewApplicatorTest.java:72)
	at com.google.fhir.analytics.view.ViewApplicatorTest.emptyForEachOrNullWithUnion(ViewApplicatorTest.java:103)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)```

Reply:
I have also verified the creation of view tables in the `VIEWS` database after adding this dependency.